libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -142,6 +143,13 @@ const AP_Param::GroupInfo AP_ICEngine::var_info[] = {
     // @Range: 0 1300
     AP_GROUPINFO("STARTCHN_MIN", 16, AP_ICEngine, start_chan_min_pwm, 0),
 
+    // @Param: REDLINE_RPM
+    // @DisplayName: RPM of the redline limit for the engine
+    // @Description: This is the maximum RPM for the engine provided by the manufacturer. A value of 0 disables this feature. See ICE_OPTIONS to enable or disable the governor.
```

Comment:
We can remove the "This is" part of the description... just jump straight to the content so perhaps, "maximum RPM for the engine provided by the manufacturer"

I think we should make it a bit more clear what the difference is between setting this value to zero and setting the option bit.

Reply:
It would be good to add // @Units: RPM

Reply:
Fixed